### PR TITLE
fix: profile dropdown in group page

### LINKED ISSUE #2739: No profile dropdown on Group page
**Describe the bug**
Clicking on profile name on Group page does not trigger the dropdown

**To Reproduce**
1. Create a group
2. On the group page, click on your profile name on top right

**Expected behavior**
Must have a dropdown
Screenshot from 2021-12-25 12-04-49

### PR DESCRIPTION
**Pull request by h2o-Noob**
Fixes #2739

#### Describe the changes you have made in this PR -
fixed profile drop-down issue on groups page by rearranging file order in groups_controller page.

### Screenshots of the changes (If any) -
**before**
ss before pofile

**after**
ss after profile

Note: Please check **Allow edits from maintainers.** if you would like us to assist in the PR.

### CODE REVIEW

**Comment by h2o-Noob** [comment]
Sorry for the delay in submitting the PR, I had my practical exams

**Comment by h2o-Noob** [comment]
@tachyons sir please review

File: app/views/layouts/_header.html.erb [hunk 1]
```diff
@@ -93,4 +93,18 @@
   if (window.location.href.includes('search')) {
       searchBar.style.display = "block";
   }
+  <script type="text/javascript">
```
Comment (tachyons) [inline app/views/layouts/_header.html.erb:96]: This seems like a workaround, Can we do this without js ?

Comment (h2o-Noob) [reply app/views/layouts/_header.html.erb:96]: Sir any suggestions on what other approach can be used? At the moment I can only think of this fix.

Comment (vedant-jain03) [reply app/views/layouts/_header.html.erb:96]: @h2o-Noob I reproduce the error and I think the error lies in the ordering of files in https://github.com/CircuitVerse/CircuitVerse/blob/7cc19ee5f624f713005b8b262bc292c9d8a262c2/app/javascript/controllers/groups_controller.js#L1

I think this ordering show works as it is working for me.
```
import { Controller } from 'stimulus';
import * as $ from 'jquery/dist/jquery';
import 'js/jquery.js';
import 'bootstrap/dist/js/bootstrap.min.js';
import 'bootstrap/dist/js/bootstrap';
import 'bootstrap-tagsinput/dist/bootstrap-tagsinput.js';
```

WDYT @tachyons ?

**Comment by h2o-Noob** [comment]
Yes this sollution is better, Thank you @vedant-jain03 for your help!

**Comment by h2o-Noob** [comment]
@tachyons Sir shall I close this pr and make a new one with changes suggested by @vedant-jain03 ?

**Comment by vedant-jain03** [comment]
> 

You can push new commit

**Comment by vedant-jain03** [comment]
> Yes this sollution is better, Thank you @vedant-jain03 for your help!

Recheck if the solutiòn is working on your desk as well?

**Comment by h2o-Noob** [comment]
> > Yes this sollution is better, Thank you @vedant-jain03 for your help!
> 
> Recheck if the solutiòn is working on your desk as well?

yes it is!

**Comment by h2o-Noob** [comment]
> > 
> 
> You can push new commit after @tachyons review.

Okay

**Comment by h2o-Noob** [comment]
@tachyons sir please review

**Comment by tachyons** [comment]
Could  you update PR title and description as well the PR description as well ?

**Comment by h2o-Noob** [comment]
> Could you update PR title and description as well the PR description as well ?

done

**Review by vedant-jain03 — APPROVED** [review]

**Comment by tachyons** [comment]
CI is red, need to make it green before merging

**Comment by tachyons** [comment]
Fixed in https://github.com/CircuitVerse/CircuitVerse/commit/a783add80d476b18db3f8e3028c002de10234381

**Pull request closed without merging**